src/components/dialog/dialog/dialog.ts

```diff
@@ -132,49 +99,51 @@ export class SbbDialogElement extends SbbNegativeMixin(LitElement) {
   /** Emits whenever the `sbb-dialog` is closed. */
   private _didClose: EventEmitter = new EventEmitter(this, SbbDialogElement.events.didClose);
 
-  /** Emits whenever the back button is clicked. */
-  private _backClick: EventEmitter<void> = new EventEmitter(
-    this,
-    SbbDialogElement.events.backClick,
-  );
-
-  private _dialog!: HTMLDivElement;
-  private _dialogWrapperElement!: HTMLElement;
+  private _dialogTitleElement!: SbbDialogTitleElement;
+  private _dialogTitleHeight!: number;
   private _dialogContentElement!: HTMLElement;
+  private _dialogActionsElement?: SbbDialogActionsElement;
   private _dialogCloseElement?: HTMLElement;
   private _dialogController!: AbortController;
-  private _windowEventsController!: AbortController;
+  private _openDialogController!: AbortController;
   private _focusHandler = new FocusHandler();
   private _scrollHandler = new ScrollHandler();
   private _returnValue: any;
   private _isPointerDownEventOnDialog: boolean = false;
+  private _overflows: boolean = false;
+  private _lastScroll = 0;
   private _dialogId = `sbb-dialog-${nextId++}`;
 
   // Last element which had focus before the dialog was opened.
   private _lastFocusedElement?: HTMLElement;
 
   private _language = new LanguageController(this);
-  private _namedSlots = new NamedSlotStateController(this, () =>
-    setAttribute(this, 'data-fullscreen', !this._hasTitle),
-  );
 
   /**
    * Opens the dialog element.
    */
   public open(): void {
-    if (this._state !== 'closed' || !this._dialog) {
+    if (this._state !== 'closed') {
       return;
     }
     this._lastFocusedElement = document.activeElement as HTMLElement;
 
+    // Initialize dialog elements
+    this._dialogTitleElement = this.querySelector('sbb-dialog-title')!;
+    this._dialogContentElement = this.querySelector('sbb-dialog-content')!.shadowRoot!
+      .firstElementChild as HTMLElement;
+    this._dialogActionsElement = this.querySelector('sbb-dialog-actions') || undefined;
```

Comment:
Should not better this._dialogActionsElement be able to deal with null?
```suggestion
    this._dialogActionsElement = this.querySelector('sbb-dialog-actions');
```